server/index.js

```diff
@@ -18,6 +22,15 @@ app.use(bodyParser.urlencoded({ extended: false }));
 
 app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerConfig));
 
+// add worker if on production environment
+if (process.env.NODE_ENV === 'production') {
+  setInterval(async () => {
+    worker();
+  }, ms(process.env.TIMER_INTERVAL));
```

Comment:
Can we have a default `TIMER_INTERVAL` here, something like this:
```
 setInterval(async () => {
    worker();
  }, ms(process.env.TIMER_INTERVAL) || defaultTimerInterval);
```
This is so that the entire applicaiton won't crash just becuase TIMER_INTERVAL is not set in the environment variable.

Reply:
Since it is not a secret, why should timer_interval be an environment variable?

Reply:
Make sense, I'll implement the change